### DISCUSSION TITLE
Fix some bugs and added new bugs

### DISCUSSION THREAD

**Discussion by dhohirpradana** [dhohirpradana/Instagram-Automation#13] [General]
Changelogs:
- fix bugs
- clean code 
- added code improvements

<hr /><em>This discussion was created from the release <a href='https://github.com/dhohirpradana/Instagram-Automation/releases/tag/v4.0'>Fix some bugs and added new bugs</a>.</em>

**Comment by githabis** [comment]
Om, kenapa ane error pas di app.py nya ya, cannot remove result.txt
